.github/workflows/ci.yml

```diff
@@ -193,3 +202,13 @@ jobs:
               console.log("no change in memory usage detected by benchmark");
             }
         if: ${{ github.base_ref != null }}
+
+  status-check:
+    name: Status Check
+    needs: [format, clippy, web-check, build, integration-tests, runBenchmark]
+    runs-on: ubuntu-latest
```

Comment:
If we're running the build on Ubuntu & Windows, then it makes sense to run the status check on Ubuntu & Windows as well, right?

Reply:
Or maybe I just didn't read this right - does this job take into account the outcomes from the previous Windows jobs?

Reply:
Nope, it only needs to run in one place, because the `needs` section identifies the list of jobs it requires (so waits on them to finish) and the `if: failed()` causes it to only trigger if any of those other jobs failed. If they all succeed, this job doesn't meet it's precondition and is skipped, and from branch protection's perspective a skipped job is treated the same as a succeeded job.

Reply:
Got it, thanks! I'd probably name this this `Fail Workflow` or something else that implies failure, instead of the innocuous sounding "status check" then. When I see "Status Check = skipped" I panic 😜